R/activate.R

```diff
@@ -156,22 +156,27 @@ renv_activate_prompt <- function(action, library, prompt, project) {
   if (!ask)
     return(FALSE)
 
-  fmt <- lines(
-    "",
-    "This project has not yet been activated.",
-    "Activating this project will ensure the project library is used when %s() is called.",
-    "Please see `?renv::activate` for more details.",
-    ""
+  renv_activate_prompt_impl(project)
+
+
+}
+
+renv_activate_prompt_impl <- function(project = NULL) {
+  choices <- c(
+    "Activate the project, setting up a project library",
+    "Continue, using the system library",
+    "Cancel"
   )
-  writef(fmt, action)
 
-  fmt <- "Would you like to activate this project before %s() is called?"
-  question <- sprintf(fmt, action)
-  response <- ask(question, default = TRUE)
-  if (!response)
-    return(FALSE)
+  writef("This project has not yet been activated.")
+  choice <- menu(choices, "What do you want to do?", default = 2)
```

Comment:
```suggestion
  choice <- menu(choices, "What would you like to do?", default = 2)
```

Also, does menu have a 'default' argument?

Reply:
```suggestion
  choice <- menu(choices, "How would you like to proceed?", default = 2)
```

Just word-smithing; not sure if one of these is better than the other.

Reply:
Oh, I see you defined your own version of menu below.